services/graylog/docker-compose.yml

```diff
@@ -92,3 +99,7 @@ networks:
   public:
     external: true
     name: ${PUBLIC_NETWORK}
+
+configs:
+  graylog_config:
+    file: ./data/contentpacks/simcore-content-pack.json
```

Comment:
Missing newlines? It looks to me like you did not install and run pre-commit hooks, please do this, thx :P

Reply:
I have run it with precommit and add some small changes on a way